lists/cl/cl-mp.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": {
+    "en": "Public Market Suppliers and Buyer Units, Chile",
+    "local": "Proveedor y Unidad Comprador Mercado Público, Chile"
+  },
+  "url": "https://www.mercadopublico.cl/Home",
+  "description": {
+    "en": "Mercado Público is an electronic platform managed by Chilecompra, part of the Ministry of Finance, which public organizations use to make purchases.\n\nOrganizations registered as buyers or suppliers on the Mercado Público platform are assigned an identifier in the system. This identifier is used in the data from the Mercado Público OCDS API.\n\nThe use of Mercado Público identifiers is considered a temporary measure. They should only be used when no other stable identifier for a government agency is available"
+  },
+  "coverage": [
+    "CL"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "company",
+    "government_agency"
+  ],
+  "sector": [],
+  "code": "CL-MP",
+  "confirmed": false,
+  "deprecated": false,
+  "listType": "local",
+  "access": {
```

Comment:
I'm not sure that this section, from l24 to l36, is consistent. Data is available online through the api.

Reply:
I've updated `availableOnline` to `true` and moved the text from `guidanceOnLocatingIds` to `onlineAccessDetails` in https://github.com/org-id/register/pull/424/commits/ca9c0f0eec1e827ccb53925860ca46ecf386ac58

For the properties under `data`, I'd interpreted "Details of whether or not an open dataset is available for this list." as meaning whether the whole list was available, whereas in this case, all you can do is retrieve the details of an organization if you already have an identifier. Does it make sense to leave it set to `data_not_available` in that case?

Reply:
Thanks @duncandewhurst 

This is definitely an area where the schema can be improved. Whereas we have:

`Dataset access | Details of whether or not an open dataset is available for this list.`

we then have

`Data availability (was DataAccessProperties) | Is structured data available from this list in any of these formats or mechanisms?`

as you point out. I'd actually be inclined to rewrite `onlineAccessDetails` first, to briefly describe what is available online. eg. "The only access to Mercado Publico data available online is through the API, which is freely available and requires no sign up."

Then 

`  "data": { 
      "availability": [
         "data_not_available"
    ],`
should change to 
`  "data": { 
      "availability": [
         "API"
    ],`

and then most of the description that you currently have in `"onlineAccessDetails"` can go in `"dataAccessDetails"`

A lot of the above is interpreted rather than being explicitly laid out and we recognise that there will be inconsistencies across org-id. We'll be writing this up as an issue.

Reply:
I've made those updates in in https://github.com/org-id/register/pull/424/commits/738a86a7f929105e33054735aaf959ba4bbb8435